### DISCUSSION TITLE
disable cmp-git but not rest of the git functionality

### DISCUSSION THREAD

**Discussion by kimusan** [LazyVim/LazyVim#4313] [Q&A]
I  have installed the lang.git extra and that gave me some nice additions.  Unfortunately, it also added the cmp-git, which is highly annoying (pressing ":" shows a useless autocomplete list that from git history) and very slow on large repositories. 

Is there any way to disable cmp-git easily without loosing the rest of the lang.git functionality?

**Comment by dpetka2001** [comment]
```lua
{
  "petertriho/cmp-git",
  enabled = false,
},
```
should probably do it I believe?

**Reply by kimusan** [reply]
thanks. not sure why I  missed that solution.
